tests/test_deployments.py

```diff
@@ -526,14 +572,38 @@ async def test_deployment(patch_import, tmp_path):
     return d, deployment_id
 
 
-async def test_deployment_apply_updates_concurrency_limit(
-    patch_import, tmp_path, orion_client
-):
-    d = Deployment(name="TEST", flow_name="fn")
-    deployment_id = await d.apply(work_queue_concurrency=424242)
-    queue_name = d.work_queue_name
-    work_queue = await orion_client.read_work_queue_by_name(queue_name)
-    assert work_queue.concurrency_limit == 424242
+class TestDeploymentApply:
+    async def test_deployment_apply_updates_concurrency_limit(
+        self, patch_import, tmp_path, orion_client
+    ):
+        d = Deployment(name="TEST", flow_name="fn")
+        deployment_id = await d.apply(work_queue_concurrency=424242)
+        queue_name = d.work_queue_name
+        work_queue = await orion_client.read_work_queue_by_name(queue_name)
+        assert work_queue.concurrency_limit == 424242
+
+    @pytest.mark.parametrize(
+        "is_active, expected_schedule_active",
+        [(True, True), (False, False), (None, True)],
+    )
+    async def test_deployment_is_active_behaves_as_expected(
+        self, flow_function, is_active, expected_schedule_active, orion_client
+    ):
+        d = await Deployment.build_from_flow(
+            flow_function,
+            name="foo",
+            tags=["A", "B"],
+            description="foobar",
+            version="12",
+            is_schedule_active=is_active,
+        )
+        assert d.flow_name == flow_function.name
+        assert d.name == "foo"
+        assert d.is_schedule_active == is_active
```

Comment:
Nit: we should probably use `is` for comparisons with this field

Reply:
Confirming my understanding - this is because we're using a bool here?

Reply:
Yep!